### DISCUSSION TITLE
Can you enable discussion on https://github.com/qiniu/qmgo? Thanks!

### DISCUSSION THREAD

**Discussion by jiangz222** [community/community#64] [Discussions] [answered]
Can you enable discussion on https://github.com/qiniu/qmgo?
Thanks!

**Comment by Neilblaze** [comment, marked as answer]
@jiangz222 drop a message to @ernest-phillips over [here](https://github.community/t/can-one-apply-for-github-discussions-beta/121120/442) & he'll definitely help you adding discussion feature on https://github.com/qiniu/qmgo

Thanks :smiley:
